### PR TITLE
feat: rework liabilities usage

### PR DESCRIPTION
**Pull request by chumaknadya**

### CODE REVIEW

File: app/internal/liability/liability.go [hunk 1]
```diff
@@ -10,79 +10,68 @@ import (
...
+	ErrEmptyByteArray         = errors.New("liability: empty byte array")
+	ErrNonExistingLiabilities = errors.New("liability: liabilities for such participant doesn't exist")
+	ErrNoReqLiability         = errors.New("liability: liability with REQ type doesn't exist")
+	ErrInvalidOperation       = errors.New("liability: given amount is bigger than actual amount")
 )
 
 type Asset string
 
-// Liability represents information about asset and amount of that asset
-type Liability struct {
```
Comment (chumaknadya) [inline app/internal/liability/liability.go:22]: let's rename it to liabilitites (plural)
but will have naming for functions like `AddLiability` (with single), cause every time we add only one

File: app/internal/liability/liability.go [hunk 2]
```diff
@@ -93,132 +82,76 @@ func (l *Liability) AddRevertLiability(asset Asset, amount decimal.Decimal) erro
...
-func (ls *LiabilityState) AddRequestLiability(from, to uint, asset Asset, amount decimal.Decimal) error {
-	liability := NewLiability()
-	liability.AddRequestLiability(asset, amount)
-	ls.AddLiability(from, to, liability)
-
-	return nil
-}
+// AddAcknowledgeLiability adds new acknowledge liability to liabilities state.
+func (ls LiabilitiesState) AddAcknowledgeLiability(from, to uint, asset Asset, amount decimal.Decimal) error {
+	_, found := ls[from][to]
```
Comment (chumaknadya) [inline app/internal/liability/liability.go:115]: here, there is no reason to execute sth if there are no `ls[from][to]`
it means that map is empty, and it 100% there is no `req` type which we need to do `revert`/`acknowledge`

File: app/internal/liability/liability.go [hunk 3]
```diff
@@ -93,132 +82,76 @@ func (l *Liability) AddRevertLiability(asset Asset, amount decimal.Decimal) erro
...
-func (ls *LiabilityState) AddAcknowledgeLiability(from, to uint, asset Asset, amount decimal.Decimal) error {
-	if ls.State[from] != nil && ls.State[from].Liabilities[to] != nil {
-		liability := ls.State[from].Liabilities[to]
-		err := liability.AddAcknowledgeLiability(asset, amount)
-		if err != nil {
-			return nil
-		}
-	} else {
-		return ErrNonExistingLiability
+	err := ls[from][to].AddAcknowledgeLiability(asset, amount)
```
Comment (chumaknadya) [inline app/internal/liability/liability.go:120]: i prefer to have operations connected to liability inside liability struct

File: app/internal/liability/liability.go (hunk 1 above)
Comment (lckmidnight) [reply app/internal/liability/liability.go:22]: perfect!

**Review by lckmidnight — APPROVED** [review]

**Pull request merged**